sdks/python/build.gradle

```diff
@@ -197,13 +197,13 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
 //    ./gradlew :beam-sdks-python:portableWordCount -PjobEndpoint=localhost:8099
 //
 task portableWordCount {
-  dependsOn portableWordCountTask('portableWordCountExample', project.hasProperty("streaming"))
+  dependsOn portableWordCountTask('portableWordCountExample', project.hasProperty("streaming"), project.hasProperty("crossLanguage"))
 }
 
-def portableWordCountTask(name, streaming) {
+def portableWordCountTask(name, streaming, crossLanguage) {
   tasks.create(name) {
     dependsOn = ['installGcpTest']
-    mustRunAfter = [':beam-runners-flink_2.11-job-server-container:docker', ':beam-sdks-python-container:docker']
+    mustRunAfter = [':beam-runners-flink_2.11-job-server-container:docker', ':beam-sdks-python-container:docker', ':beam-sdks-java-container:docker', ':beam-runners-core-construction-java:jarTestExpansionService']
```

Comment:
Conditionally add these cross-language dependencie?

Reply:
Thanks for your comment! AFAIK, `mustRunAfter` just defines the order of task execution in a parallel build, meaning that the task won't automatically invoke `mustRunAfter` tasks like traditional dependencies (`dependsOn`) when it's executed. I think there's no harm in having cross-language related tasks on `mustRunAfter` list. WDYT?

Reply:
OK.